### DISCUSSION TITLE
How to store a large number of images in a repository?

### DISCUSSION THREAD

**Discussion by TungusSs** [vercel/next.js#66949] [Help]
### Summary

Hello! I'm asking you to help me with a problem.

I am currently developing a web portal for a game project, and I have to use images for my Gallery. However, there are a lot of images there, almost 100 images in .webp format. Because of this amount of images, my repository started to weigh almost 1 GB. 

Is it possible for me to reduce this size in any way? Somehow store them on Vercel itself or in a server component? There is either no information on the Internet, or it is outdated, or the discussion does not reveal the algorithm of actions how to do it.

Please help!

**Comment by icyJoseph** [comment]
Why not store these images in an image service provider? like cloudinary? 

Also, often, people use https://git-lfs.com/ to decouple these assets from the mainline repository
